backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt

```diff
@@ -34,6 +33,7 @@ public class BpkColors private constructor(
     public val line: Color,
     public val lineOnDark: Color,
     public val scrim: Color,
+    public val overlay: Color,
```

Comment:
this file is auto-generated from foundations - you have to add the tokens in backpack foundations first: https://github.com/Skyscanner/backpack-foundations

Reply:
oh that's something new to me, lemme clone that repo and add them there then

Reply:
give me a shout if you need some help!